homeassistant/components/zwave_js/services.py

```diff
@@ -418,3 +436,9 @@ async def async_multicast_set_value(self, service: ServiceCall) -> None:
 
         if success is False:
             raise SetValueFailed("Unable to set value via multicast")
+
+    async def async_ping(self, service: ServiceCall) -> None:
+        """Ping node(s)."""
+        nodes: set[ZwaveNode] = service.data[const.ATTR_NODES]
+        for node in nodes:
+            await node.async_ping()
```

Comment:
Should we await in sequence or use `asyncio.gather` to await concurrently?

Reply:
I switched it but I think this is a better question for Al. Will ask him and adjust accordingly.

Reply:
Spoke to Al and he said it doesn't matter which approach we use. I think we should leave it as you suggested and let the driver handle it